config/config.go

```diff
@@ -475,3 +467,21 @@ type GasScheduleByEpochs struct {
 type GasScheduleConfig struct {
 	GasScheduleByEpochs []GasScheduleByEpochs
 }
+
+// ConfigurationFileName holds all configuration filenames used to start the node
+type ConfigurationFileName struct {
```

Comment:
maybe rename to `ConfigurationPathsHolder` or something similar because we also have directories in this struct

Reply:
done